tests/instancetype_test.go

```diff
@@ -335,6 +339,7 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 				Name: preference.Name,
 				Kind: instancetypeapi.SingularPreferenceResourceName,
 			}
+			vm.Spec.Template.Spec.TerminationGracePeriodSeconds = nil
```

Comment:
Can you add this into `removeResourcesAndPreferencesFromVMI` and remove the changes to the comment?